superset/migrations/versions/58df9d617f14_add_on_saved_query_delete_tab_state_.py

```diff
@@ -17,14 +17,14 @@
 """add_on_saved_query_delete_tab_state_null_constraint"
 
 Revision ID: 58df9d617f14
-Revises: 6766938c6065
+Revises: 8b841273bec3
 Create Date: 2022-03-16 23:24:40.278937
 
 """
 
 # revision identifiers, used by Alembic.
 revision = "58df9d617f14"
-down_revision = "6766938c6065"
+down_revision = "8b841273bec3"
```

Comment:
@diegomedina248 this is going to be tricky to change a down revision for a migration that has already merged, especially for people who have already run this migration. Is there any reason why the new file can't run after this one?

Reply:
@eschutho no, you're absolutely right! I solved the migration conflict incorrectly
Should be ok no, thanks.